gradle/libs.versions.toml

```diff
@@ -1,52 +1,52 @@
 [versions]
 kotlin-version = "1.8.22"
-kotlinx-html-version = "0.9.1"
+kotlinx-html-version = "0.11.0"
```

Comment:
Seems this may also be causing some problems with building common artifacts...

```
java.lang.IllegalStateException: e: Could not find "/mnt/agent/work/a5ccb8c76251a2e/ktor/ktor-server/ktor-server-plugins/ktor-server-html-builder/build/kotlinTransformedMetadataLibraries/jvmAndNixMain/org.jetbrains.kotlinx-kotlinx-html-0.11.0-commonMain-W4uI5A.klib" in [/home/teamcity/.local/share/kotlin/daemon]
```

Reply:
yep